src/modules/configs/config.py

```diff
@@ -0,0 +1,48 @@
+import os
+from json import loads
+from typing import Union
+
+from jsonschema import validate, ValidationError
+
+from ingestation.modules.global_vars import APPLICATION_DIR
+
+
+def validate_json(conf: dict, schema_file) -> None:
+    validate(instance=conf, schema=getattr(schema_file, "schema"))
+
+
+class Conf:
+    """Class for parsing configuration files"""
+
+    def __init__(self, conf_path: Union[bool, str], conf_type: str, schema_file):
+        self.conf_type = conf_type
+        self.conf_path = self.get_conf_path(conf_path)
+        self.schema_file = schema_file
+
+    def parse_conf(self) -> Union[bool, dict]:
+        conf = self.load_conf()
+
+        try:
+            validate_json(conf, self.schema_file)
+        except ValidationError as ex:
+            raise ValidationError(
+                f"[CRITICAL]: {self.conf_type.capitalize()} config validation failed.") \
```

Comment:
We should mention sth like *Review application config at {self.conf_path} and correct errors.*